build-virtualbox.ps1

```diff
@@ -9,14 +9,13 @@ param(
   [switch]$debug
 )
 
-. Tests/powershell-helpers.ps1
+$path = Join-Path -Path $PSScriptRoot -ChildPath "powershell-helpers.ps1"
```

Comment:
Probably dont need this trick here - we should always be in the same folder as the command we're running.
We just need it in the other script, as that gets included from other folders